### ISSUE TITLE
Will you accept a PR updating the Dependencies?

### ISSUE THREAD

**Issue by nelsonic** [anvilresearch/connect#300]
At present a few dependencies are out of date: [![Dependency Status](https://david-dm.org/anvilresearch/connect.svg)](https://david-dm.org/anvilresearch/connect) can we submit a PR updating them?

**Comment by christiansmith** [comment]
Just took a look using [david](https://github.com/alanshaw/david). 

```
$ david
dependencies

┌───────────────────┬─────────────────────┬────────┐
│ Name              │ Package             │ Latest │
├───────────────────┼─────────────────────┼────────┤
│ ldapjs            │ mcavage/node-ldapjs │ 1.0.0  │
├───────────────────┼─────────────────────┼────────┤
│ qs                │ ^5.0.0              │ 6.0.1  │
├───────────────────┼─────────────────────┼────────┤
│ passport-saml     │ ^0.13.0             │ 0.14.0 │
├───────────────────┼─────────────────────┼────────┤
│ passport-ldapauth │ ^0.3.0              │ 0.5.0  │
└───────────────────┴─────────────────────┴────────┘

npm install --save ldapjs@1.0.0 qs@6.0.1 passport-saml@0.14.0 passport-ldapauth@0.5.0

devDependencies

┌──────┬─────────┬────────┐
│ Name │ Package │ Latest │
├──────┼─────────┼────────┤
│ nock │ ^2.2.0  │ 3.4.1  │
└──────┴─────────┴────────┘

npm install --save-dev nock@3.4.1
```

I can accept an update to nock and qs, for sure.

The LDAP and SAML related packages require thorough testing against live instances with the new library versions and we don't really have time to go there before year end unless @vsimonian or someone else wants to jump back in and help to verify.

**Comment by nelsonic** [comment]
@christiansmith should I update the version of the package to [0.1.59](https://github.com/anvilresearch/connect/blob/3fa90640ca8af35c94f62d55481d97f03d9264a2/package.json#L4) in `package.json` ?

**Comment by christiansmith** [comment]
Please do not update the package version. We use grunt-release to automate releases, and this increments the version in package.json. We have other outstanding work to wrap up before doing another release.

**Comment by nelsonic** [comment]
@christiansmith I have not updated the package version in the PR. 
But did update the version of Node.js in `.travis.yml` as discussed in #302

**Comment by nelsonic** [comment]
Closed by #301
